### PR TITLE
Ensure flake8-encoding

### PR DESCRIPTION
**Pull request by harikishen**
@garbas

### CODE REVIEW

**Comment by garbas** [comment]
@harikishen this looks good also tests pass for releng-treestatus, before we merged this lets apply this to all of the python projects

**Comment by harikishen** [comment]
@garbas I've done the check in all projects.

File: src/releng_archiver/requirements_frozen.txt [hunk 1]
```diff
@@ -65,3 +65,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 Werkzeug==0.12.2
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/releng_archiver/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/shipit_signoff/requirements_frozen.txt [hunk 2]
```diff
@@ -69,3 +69,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 Werkzeug==0.12.2
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/shipit_signoff/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/releng_clobberer/requirements_frozen.txt [hunk 3]
```diff
@@ -65,3 +65,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 Werkzeug==0.12.2
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/releng_clobberer/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/releng_mapper/requirements_frozen.txt [hunk 4]
```diff
@@ -65,3 +65,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 Werkzeug==0.12.2
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/releng_mapper/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/releng_tooltool/requirements_frozen.txt [hunk 5]
```diff
@@ -66,3 +66,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 Werkzeug==0.12.2
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/releng_tooltool/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/shipit_bot_uplift/requirements_frozen.txt [hunk 6]
```diff
@@ -61,3 +61,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 whatthepatch==0.0.5
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/shipit_bot_uplift/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/shipit_code_coverage/requirements_frozen.txt [hunk 7]
```diff
@@ -49,3 +49,4 @@ uritemplate==3.0.0
 urllib3==1.21.1
 whatthepatch==0.0.5
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/shipit_code_coverage/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/shipit_pipeline/requirements_frozen.txt [hunk 8]
```diff
@@ -65,3 +65,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 Werkzeug==0.12.2
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/shipit_pipeline/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/shipit_pulse_listener/requirements_frozen.txt [hunk 9]
```diff
@@ -26,3 +26,4 @@ structlog==17.2.0
 taskcluster==1.3.3
 urllib3==1.21.1
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/shipit_pulse_listener/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/shipit_risk_assessment/requirements_frozen.txt [hunk 10]
```diff
@@ -29,3 +29,4 @@ taskcluster==1.3.3
 urllib3==1.21.1
 whatthepatch==0.0.5
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/shipit_risk_assessment/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/shipit_static_analysis/requirements_frozen.txt [hunk 11]
```diff
@@ -49,3 +49,4 @@ uritemplate==3.0.0
 urllib3==1.21.1
 whatthepatch==0.0.5
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/shipit_static_analysis/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/shipit_taskcluster/requirements_frozen.txt [hunk 12]
```diff
@@ -65,3 +65,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 Werkzeug==0.12.2
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/shipit_taskcluster/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

File: src/shipit_uplift/requirements_frozen.txt [hunk 13]
```diff
@@ -66,3 +66,4 @@ urllib3==1.21.1
 wcwidth==0.1.7
 Werkzeug==0.12.2
 yarl==0.10.2
+flake8-coding
```
Comment (garbas) [inline src/shipit_uplift/requirements_frozen.txt]: this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``

**Comment by harikishen** [comment]
@garbas i've modified the requirements_frozen.txt files

**Pull request merged**